src/ContactManager/Contacts.cs

```diff
@@ -0,0 +1,47 @@
+﻿// <copyright file="Contacts.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+using System.Security.Cryptography.X509Certificates;
+
+namespace ContactManager
+{
+    public class Contacts
+    {
+        public string name;
```

Comment:
The variables should be private as the Property is exposed as public.

Reply:
Changed as required